datasets/harem/harem.py

```diff
@@ -0,0 +1,279 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""HAREM dataset"""
+
+from __future__ import absolute_import, division, print_function
+
+import json
+import logging
+from typing import List, Tuple
+
+from transformers.tokenization_utils import _is_punctuation, _is_whitespace
```

Comment:
Maybe we can avoid the dependency on transformers just for those two functions ?

Feel free to copy paste them in the script from
https://github.com/huggingface/transformers/blob/master/src/transformers/tokenization_utils.py#L77-L89
and
https://github.com/huggingface/transformers/blob/master/src/transformers/tokenization_utils.py#L53-L62

Reply:
Hi @lhoestq, I've just made the adjustments you've proposed